java/src/jmri/jmrix/powerline/dmx512/Bundle.java

```diff
@@ -0,0 +1,99 @@
+package jmri.jmrix.powerline.dmx512;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.CheckReturnValue;
+import javax.annotation.CheckForNull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ *
+ * Convention is to provide a subclass of this name in each package, working off
+ * the local resource bundle name.
+ *
+ * @author Bob Jacobsen Copyright (C) 2012
+ * @author Ken Cameron Copyright (C) 2023
+ * @since 3.3.1
+ */
+public class Bundle extends jmri.jmrix.powerline.Bundle {
+
+    @CheckForNull
+    private static final String name = null; // No local resources
```

Comment:
Since you have added the file `jmri/jmrix/powerline/dmx512/PowerlineActionListBundle.properties`, it might be good to refer it on this line even if it's empty. If someone later adds a property to the file, he might be confused if it doesn't work.

Reply:
Yes the intent is to have an empty menu for Powerline when DMX is the type. But this still gives the common ones of monitor and send command, both of which don't do anything in DMX. So I need to find some other way. The other connection types all have that as null.

Reply:
If you don't intend to use [jmri/jmrix/powerline/dmx512/PowerlineActionListBundle.properties](https://github.com/JMRI/JMRI/pull/11754/files#diff-14c4ff4d10e114b6bcb6116273e6c1605feb847bfdfe547dd297edd1b77a00b8), I suggest that you delete it.